packages/forma-36-website/src/pages/foundation/typography/index.mdx

```diff
@@ -2,4 +2,222 @@
 title: 'Typography'
 ---
 
+import fontBaseTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-base';
+import fontSizeTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-size';
+import fontWeightTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-weight';
+import lineHeightTokens from '../../../../../forma-36-tokens/src/tokens/typography/line-height';
+import fontStackTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-stack';
+import letterSpacingTokens from '../../../../../forma-36-tokens/src/tokens/typography/letter-spacing';
+
+import {
+  Heading,
+  Subheading,
+  Table,
+  TableHead,
+  TableBody,
+  TableCell,
+  TableRow,
+} from '@contentful/forma-36-react-components';
+
 # Typography
+
+## Font base
+
+Our font base is used in combination with the REM font sizes below
+
+<Table className="f36-margin-bottom--l">
+  <TableHead>
+    <TableRow>
+      <TableCell>Token</TableCell>
+      <TableCell>CSS Variable</TableCell>
+      <TableCell>Value</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    {Object.keys(fontBaseTokens).map(token => {
```

Comment:
Markdown table syntax works and would be preferred but I guess you lose the ability to build up rows dynamically.

Reply:
yes i'd rather use the table to dynamically build it.. will be less error prone